data/json/items/tool/explosives.json

```
@@ -1015,12 +1015,13 @@
     "flags": [ "BOMB", "TRADER_AVOID", "NO_REPAIR" ]
   },
   {
-    "id": "pipebomb",
+    "id": "bootleg_pipebomb",
     "type": "TOOL",
     "category": "weapons",
-    "name": { "str": "pipe bomb" },
-    "description": "This is a section of a pipe filled with explosive materials.  Use this item to light the fuse, which gives you five turns to get away from it before it detonates.  You'll need a lighter or some matches to use it.",
-    "weight": "730 g",
+    "name": { "str": "bootleg pipe bomb" },
+    "looks_like": "pipebomb",
+    "description": "This is a section of a pipe filled with military explosives.  Use this item to pull the pin, which gives you five turns to get away from it before it detonates.",
```

```suggestion
    "description": "This is a section of a pipe filled with military explosives.  Use this item to pull the pin, which gives you six seconds to get away from it before it detonates.",
```
"Turns" are both gamey and generally incorrect. (player's speed isn't constant) Also, the screenshots show 6 seconds.

---

Absolutely right, changed them all to six seconds.